js/controller.js

```diff
@@ -331,6 +331,12 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       }
     },
 
+    /**
+     * @description
+     * Should be used with deviceorientation event listener.
+     * Uses for video 360 on mobile devices for setting necessary coordinates (relevant with start device orientation)
+     * @param e - event
```

Comment:
Can we change this to the format we use in other JSDoc.  @param {object} e The event object.

Reply:
\+